app/models/spree/payment_decorator.rb

```diff
@@ -32,11 +32,7 @@ def adjustment_label
 
     # This is called by the calculator of a payment method
     def line_items
-      if order.complete? && Spree::Config[:track_inventory_levels]
-        order.line_items.select { |li| inventory_units.pluck(:variant_id).include?(li.variant_id) }
```

Comment:
why is this not needed anymore?

Reply:
this is in the scope of adjustments. see calculator_decorator.line_items_for..
shipment like payment has one adjustment.
this was incorrectly copied from shipment where there are inventory_units to be tracked.
payment doesnt have inventory_units ;-)

Reply:
:scream: Your're becoming our CAO, chief adjustments officer :trollface:

Reply:
ahaha, CAO means dog in Portuguese 😆